pkg/core/native/nativehashes/gen.go

```diff
@@ -0,0 +1,82 @@
+//go:build ignore
+
+// This program generates hashes.go. It can be invoked by running
+// go generate.
+package main
+
+import (
+	"log"
+	"os"
+	"text/template"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/native/nativenames"
+	"github.com/nspcc-dev/neo-go/pkg/core/state"
+)
+
+// srcTmpl is a nativehashes package template.
+const srcTmpl = `// Code generated by "go generate go run gen.go"; DO NOT EDIT.
+
+//go:generate go run gen.go
+
+// package nativehashes contains hashes of all native contracts in their LE and Uint160 representation.
+package nativehashes
+
+import "github.com/nspcc-dev/neo-go/pkg/util"
+
+// Hashes of all native contracts in the LE string representation.
+const(
+{{- range .Natives }}
+	// {{ .Name }}StringLE is a hash of native {{ .Name }} contract in the LE string representation.
+	{{ .Name }}StringLE = {{ printf "%q" .HashLE }}
```

Comment:
Do you need it?

Reply:
I thought that maybe someone can make use of strings, but you're right, it seems to be extensive, we don't need it at all.